### ISSUE TITLE
Add Google Cloud Storage support

### ISSUE THREAD

**Issue by matticoli** [jazzhistorydatabase/jhdb-cms#31]
Some functionality dependent on #30 

Node site users will select files from their computer to upload, which will be saved in a GCP cloud bucket and linked in firebase. This should be designed similarly to the `dropbox.js` module

**Comment by matticoli** [comment]
No longer needed. See #48

**Comment by matticoli** [comment]
Consolidate into #48
